src/api.js

```diff
@@ -21,9 +21,43 @@ export async function getReviews(query = '') {
 }
 
 export async function getScreenings(query = '') {
-    const res = await fetch(APIData + "/screenings" + query);
-    const content = await res.json();
-    return content.data;
-}
+    const res = await fetch(APIData + "/screenings?populate=movie");
+    //Query string ?filters=short. Get screenings the comming 5 days, max 10 screenings.
+    if (query.filters === "short" || query === "/homepage") {
+        const content = await res.json();
+        let data = content.data;
+        let screeningArray = [];
+        let filteredArray = [];
+        let shortArray = [];
+
+        const todaysDate = new Date().toISOString();
+        const fiveDays = new Date();
+
+        fiveDays.setDate(new Date().getDate() + 4);
+
+        const fiveDaysFilter = fiveDays.toISOString();
+
+        for (let i = 0; i < data.length; i++) {
+            screeningArray.push(data[i]);
+        }
 
+        filteredArray = screeningArray.filter(data => { 
+            return data.attributes.start_time > todaysDate
+             && data.attributes.start_time < fiveDaysFilter
+        });
 
+        for (let j = 0; j < filteredArray.length; j++) {
+            if (shortArray.length <= 9 ) {
+                shortArray.push(filteredArray[j]);
+            }
+            else {
+                break;
+            }
+        }
+        return shortArray;
+    }
```

Comment:
As you mentioned maybe it will be easier/cleaner to seperate the filter into another function like "filterComingScreenings" 👍 Overall everything looks really good! Great 💯